app/src/main/res/drawable/general_item_background_border_grey.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <solid android:color="@color/component_color_profile_rename_activity_secondary_background_color" />
```

Comment:
Don't use `profile, activity, background` like terms in naming for this file.

`component_color_general_item_background_solid_color`

Reply:
done